addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -236,7 +240,7 @@ private synchronized void refreshData() {
 		try {
 			refreshDeviceData();
 			if (connectionEstablished) {
-				updateStatus(ThingStatus.ONLINE);
+				if (!getThing().getStatus().equals(ThingStatus.ONLINE)) updateStatus(ThingStatus.ONLINE);
```

Comment:
This check should not be done. It should be set to ONLINE, even if it already is online - but we can be sure we have a more recent timestamp for this update.